src/input/cursor.c

```diff
@@ -663,8 +658,7 @@ _cursor_update_focus(struct server *server)
 	}
 
 	double sx, sy;
-	cursor_update_common(server, &ctx, msec(&now),
-		/*cursor_has_moved*/ false, &sx, &sy);
+	cursor_update_common(server, &ctx, /*cursor_has_moved*/ false, &sx, &sy);
```

Comment:
Looks like the first two lines (now + clock_gettime) of the function can now be removed.